### PR TITLE
Migrate various tests & utilities to the new Galley metadata code.

### PR DESCRIPTION
**Pull request by ozevren**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: tests/integration/galley/conversion_test.go [hunk 1]
```diff
@@ -57,9 +57,9 @@ func TestConversion(t *testing.T) {
 						// Do init for the first set. Use Meshconfig file in this set.
 						if i == 0 {
 							if fset.HasMeshConfigFile() {
-								mc, err := fset.LoadMeshConfigFile()
-								if err != nil {
-									t.Fatalf("Error loading Mesh config file: %v", err)
+								mc, er := fset.LoadMeshConfigFile()
+								if er != nil {
```
Comment (ayj) [inline tests/integration/galley/conversion_test.go]: Is this standard compiler/linter complaining about these?

Comment (ozevren) [reply tests/integration/galley/conversion_test.go]: This is Goland rules mostly. It makes harder to go through files and fix things, so I decided to clean things up a bit as I go.

**Pull request merged**